governance/xc_admin/packages/xc_admin_frontend/components/InstructionViews/WormholeInstructionView.tsx

```diff
@@ -130,26 +135,48 @@ export const WormholeInstructionView = ({
                             key={index}
                             className="flex justify-between border-t border-beige-300 py-3"
                           >
-                            <div>{key}</div>
-                            {parsedInstruction.args[key] instanceof
-                            PublicKey ? (
-                              <CopyPubkey
-                                pubkey={parsedInstruction.args[key].toBase58()}
-                              />
-                            ) : typeof instruction.args[key] === 'string' &&
-                              isPubkey(instruction.args[key]) ? (
-                              <CopyPubkey
-                                pubkey={parsedInstruction.args[key]}
-                              />
+                            {key === 'lamports' &&
+                            typeof parsedInstruction.args[key] === 'bigint' ? (
+                              <>
+                                <div>{'PGAS'}</div>
```

Comment:
Is this case only possible on pyth clusters? If not maybe we should show PGAS/SOL based on the cluster.